openmicroscopy/openmicroscopy
# Config big image zoom

the same as https://github.com/openmicroscopy/openmicroscopy/pull/3627/

Test with local server:
 - Import a big (pyramid) image
 - View image in fullviewer. Expected behaviour: Zoom level is set automatically to a reasonable value.
 - Set the zoom level property to zero with ```omero config set omero.client.viewer.initial_zoom_level 0``` and restart the server
 - Open the image in fullviewer; the initial zoom level should be zoomed out as far as possible (make sure it's the same for Web and Insight)
 - Change the property with ```omero config set omero.client.viewer.initial_zoom_level 5``` and restart the server
 - Open the image again in fullviewer; now the initial zoom level should be 100% 

Make sure Web and Insight show the same

NOTE: please make sure you log out before changing omero config value. Server config is loaded on login once
